src/app/frontend/deploy/createnamespace_controller.js

```diff
@@ -58,6 +58,7 @@ export default class NamespaceDialogController {
   cancel() { this.mdDialog_.cancel(); }
 
   /**
+   * @export
    * Creates new namespace based on the state of the controller.
    */
```

Comment:
Everywhere else `@export` and `@private` is the last line of comment so maybe switch this to be consistent.